pkg/antctl/raw/multicluster/init.go

```diff
@@ -0,0 +1,169 @@
+		Example: initExample,
+		RunE:    initRunE,
+	}
+
+	o := initOptions{}
+	initOpts = &o
+	command.Flags().StringVarP(&o.namespace, "namespace", "n", "", "Antrea Multi-cluster Namespace")
+	command.Flags().StringVarP(&o.clusterSet, "clusterset", "", "", "ClusterSet ID of the leader cluster")
+	command.Flags().StringVarP(&o.clusterID, "clusterid", "", "", "ClusterID of the leader cluster")
+	command.Flags().StringVarP(&o.secret, "secret", "", "", "If specified, a Secret will be created for joining the member cluster")
+	command.Flags().StringVarP(&o.output, "output-file", "o", "", "Output file to save the member cluster join config")
+
+	return command
+}
+
+func initRunE(cmd *cobra.Command, args []string) error {
+	if err := initOpts.validate(); err != nil {
+		return err
+	}
+	k8sClient, err := common.NewClient(cmd)
+	if err != nil {
+		return err
+	}
+	createdRes := []map[string]interface{}{}
+	var createErr error
+	defer func() {
+		if createErr != nil {
+			fmt.Fprintf(cmd.OutOrStderr(), "Failed to init the Antrea Multi-cluster. Deleting the created resources\n")
+			if err := common.Rollback(cmd, k8sClient, createdRes); err != nil {
+				fmt.Fprintf(cmd.OutOrStdout(), "Failed to rollback: %v\n", err)
+			}
+		}
+	}()
+	createErr = common.CreateClusterClaim(cmd, k8sClient, initOpts.namespace, initOpts.clusterSet, initOpts.clusterID, &createdRes)
+	if createErr != nil {
+		return createErr
+	}
+	createErr = common.CreateClusterSet(cmd, k8sClient, initOpts.namespace, initOpts.clusterSet, "", "", "", initOpts.clusterID, initOpts.namespace, &createdRes)
+	if createErr != nil {
+		return createErr
+	}
+
+	var file *os.File
+	if initOpts.output != "" {
+		if file, err = os.OpenFile(initOpts.output, os.O_WRONLY|os.O_CREATE|os.O_TRUNC|os.O_APPEND, 0644); err != nil {
+			fmt.Fprintf(cmd.OutOrStderr(), "Failed to open file %s: %v\n", initOpts.output, err)
+		}
+	}
+	defer file.Close()
+
+	if err := outputConfig(cmd, file); err != nil {
+		return err
+	}
+	if initOpts.secret != "" {
+		if createErr = common.CreateMemberToken(cmd, k8sClient, initOpts.secret, "", "", initOpts.namespace, file, &createdRes); createErr != nil {
+			fmt.Fprintf(cmd.OutOrStderr(), "Failed to create secret: %v\n", createErr)
+			return createErr
+		}
+	}
+
+	fmt.Fprintf(cmd.OutOrStdout(), "Successfully initialized the leader cluster. Run \"antctl mc join\" on a member cluster to join the ClusterSet\n")
+
+	return nil
+}
+
+func outputConfig(cmd *cobra.Command, file *os.File) error {
+	if file == nil {
+		return nil
+	}
+	kubeconfig, err := raw.ResolveKubeconfig(cmd)
+	if err != nil {
+		return err
+	}
+
+	config := map[string]interface{}{
+		"apiVersion": "multicluster.antrea.io/v1alpha1",
+		"kind":       "ClusterSetConfig",
+		"leaderCluster": clusterOptions{
+			Namespace: initOpts.namespace,
+			ClusterID: initOpts.clusterID,
+			Secret:    initOpts.secret,
+			APIServer: kubeconfig.Host,
+		},
+		"memberCluster": clusterOptions{
+			Namespace: "kube-system",
```

Comment:
I would keep it "member-cluster-namespace" and let users change it. Anyway them need to change the member cluster ID.

If you want to use "kube-system", at least use a constant.

Reply:
Probably we should set `ClusterID` to "" too in the generated yaml, which forces the users to set it in "join".

Reply:
Changed the two variable to ""  now.